frontend/src/scenes/paths/NewPaths.js

```diff
@@ -230,11 +230,11 @@ export function NewPaths({ dashboardItemId = null, filters = null, color = 'whit
             })
     }
 
-    const dropOffValue = (pathItemCard) => {
+    const getDropOffValue = (pathItemCard) => {
         return pathItemCard.value - pathItemCard.sourceLinks.reduce((prev, curr) => prev + curr.value, 0)
     }
 
-    const completedValue = (sourceLinks) => {
+    const getCompletedValue = (sourceLinks) => {
```

Comment:
If you want this to be the number of people who reached the node, then it's `targetLinks`, or simply `pathItemCard.value`